### ISSUE TITLE
Show which account received the notification

### ISSUE THREAD

**Issue by amadejpapez** [mastodon/mastodon-ios#659]
Hello!

With multiple accounts signed in, it would be more intuitive if push notifications mentioned from which account notification is coming from.

**Comment by cooljeanius** [comment]
agreed, although given how slow the experience with receiving notifications from multiple accounts already is, I kinda worry that fixing this might worsen the situation...
